lib/tmdb/client.rb

```diff
@@ -1,40 +1,38 @@
 # frozen_string_literal: true
 
 module Tmdb
-  class Client
+  module Client
     class Error < StandardError; end
     BASE_URL = 'https://api.themoviedb.org/3'
     API_KEY = ENV['tmdb_api_key']
 
     class << self
       def movie_search(movie_title)
-        query = I18n.transliterate(movie_title).titlecase
-        data = get_parsed_movie_search_results(query)
-        not_found = "No results for '#{query}'." if data.blank?
+        data = request(:movie_search, query: movie_title)[:results]
+        not_found = "No results for '#{movie_title}'." if data.blank?
         movies = MovieSearch.parse_results(data) if data.present?
 
         OpenStruct.new(
           movie_title: movie_title,
           not_found_message: not_found,
-          query: query,
+          query: movie_title,
           movies: movies
         )
       end
 
       def get_movies_for_actor(actor_name:, page:, sort_by:)
-        person_url = url_for_person_search(actor_name)
-        person_data = request_data_from_api(person_url)&.dig(:results)&.first
+        person_data = request(:person_search, query: actor_name)[:results]&.first
 
         return OpenStruct.new(not_found_message: "No actors found for '#{actor_name}'.") if person_data.blank?
```

Comment:
Nitpick. Can we break this up instead of a one-liner?

```ruby
        return OpenStruct.new(
          not_found_message: "No actors found for '#{actor_name}'."
        ) if person_data.blank?
```